packages/qdrant-loader/src/qdrant_loader/webhooks/server.py

```
@@ -55,10 +55,22 @@
+    cognito_enabled = os.getenv("WEBHOOK_ENABLE_COGNITO_JWT", "false").lower() in (
+        "true",
+        "1",
+        "yes",
+    )
+    if not (has_global_secret or has_project_secret or cognito_enabled):
         raise RuntimeError(
-            f"{WEBHOOK_SECRET_ENV_VAR} or WEBHOOK_SECRETS must be set before "
-            "starting the webhook server."
+            "Webhook authentication is not configured. Set WEBHOOK_SECRET, "
+            "WEBHOOK_SECRETS, WEBHOOK_SECRET_<PROJECT_ID>, or enable Cognito JWT."
         )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)